src/shared/classes/MessagebarClasses.js

```diff
@@ -0,0 +1,41 @@
+import { cls } from '../cls.js';
+
+export const MessagebarClasses = (props, colors) => {
+  const {
+    rightClassName = '',
+    rightClass = '',
+    leftClassName = '',
+    leftClass,
+  } = props;
+  return {
+    base: {
+      common: 'fixed bottom-0 left-0 w-full',
```

Comment:
we don't use left/right names in classes because we support RTL, use start/end for margin/padding/inset